foreman/data_refinery_foreman/foreman/management/commands/automatic_survey.py

```diff
@@ -0,0 +1,72 @@
+"""
+Automatic survey command.
+"""
+
+from django.core.management.base import BaseCommand
+from django.template.defaultfilters import pluralize
+from django.utils import timezone
+
+from dateutil.relativedelta import relativedelta
+
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_common.models import GatheredAccession, SurveyedAccession, SurveyJob
+from data_refinery_foreman.surveyor.management.commands.surveyor_dispatcher import (
+    queue_surveyor_for_accession,
+)
+
+logger = get_and_configure_logger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("-d", "--days", help="The time window in days.", type=int, default=7)
+        parser.add_argument(
+            "-q",
+            "--quota",
+            help="The number of accessions to survey per the time window.",
+            type=int,
+            default=10,
+        )
+
+    def handle(self, *args, **options):
+        create_survey_jobs(options["days"], options["quota"])
+
+
+def create_survey_jobs(days, quota):
+    """Automatically creates survey jobs based on the passed time window requirements."""
+    surveyed_accessions = (
+        sa["accession_code"] for sa in SurveyedAccession.objects.values("accession_code")
+    )
+    gathered_accessions = [
+        ga["accession_code"]
+        for ga in GatheredAccession.objects.exclude(accession_code__in=surveyed_accessions)
+        .order_by("created_at")
+        .values("accession_code")
+    ]
+
+    time_window_survey_jobs = SurveyJob.objects.filter(
+        created_at__gt=timezone.now() - relativedelta(days=days)
+    )
+
+    remaining_quota = quota - time_window_survey_jobs.count()
```

Comment:
Could we move this logic into another function in this file that would help us look at the strategy for queuing jobs